README.md

```diff
@@ -178,7 +178,7 @@ For information on getting started, refer to the [CONTRIBUTING instructions](CON
 
 ## Release process
 
-The release process of `alizer` is very straightforward. You can create a new release [here](https://github.com/devfile/alizer/releases/new).
+An Alizer release is created each time a PR having updates on code is merged. The release process of `alizer` is very straightforward. You can create a new release [here](https://github.com/devfile/alizer/releases/new).
 
 - The _title_ of the release has to be the new version. `Alizer` follows the `v{major}.{minor}.{bugfix}` format (e.g `v0.1.0`)
 - The _description_ of the release is optional. You may add a description if there were outstanding updates in the project, not mentioned in the issues or PRs of this release.
```

Comment:
Do we need to create a tag?

Reply:
Yes I think is missing here. I've added a line which explicitly states the tag creation.